notify.c

```diff
@@ -107,6 +109,19 @@ int main(int argc, char** argv) {
 
     _Bool has_title = false;
 
+    // check for piped data
+    // should it be possible to pipe message data to this?
+    if ( !isatty(STDIN_FILENO) ) {
+        char buf[BUFSIZ];
+        char msg[BUFSIZ];
+        while (fgets(buf, sizeof buf, stdin)) {
+            strcat(msg, buf);
+        }
+        if (msg[strlen(msg)-1] == '\n') {
+            info_text = c_cfstr(msg);
+        }
+    }
+
```

Comment:
I could add in a check to see if `info_text` is still null to prevent overwriting later.

Originally I was thinking the title empty check would be sufficient, but maybe that could become a pain point? Should we just set a default title if empty? Something like set a default title when piped data is present and overwrite later if the `title` argument is supplied? That seems kind of messy though.

Reply:
Do you mean testing for this? 

I just manually ran `echo "hello, world" | ./notify -title "test notification"`

And I also created a file with a couple lines and ran `cat test.txt | ./notify -title "test notification"`

test.txt looked like:

```
hello, world
good night moon
```